PowerShell/JumpCloud Module/Public/Groups/UserGroups/Get-JCUserGroupMember.ps1

```diff
@@ -52,7 +52,7 @@ Function Get-JCUserGroupMember ()
 
                     $limitURL = "{0}/api/v2/usergroups/{1}/members" -f $JCUrlBasePath, $Group_ID
                     Write-Debug $limitURL
-                    $rawResults = Get-JCResults -Url $limitURL
+                    $rawResults = Get-JCResults -Url $limitURL -method "GET" -limit 100
```

Comment:
I'd love to see the method pulled in dynamically from the function name or similar? Just a note for us to eventually think about. Maybe we can do something similar with limit?

Reply:
I was thinking this too, however in looking at some of the Set-JC* functions, we aren't always using a PUT or a POST method, it varies as far as I can tell.

See `Set-JCSystem `and then `Set-JCSystemUser`